cmd/ctr/commands/commands.go

```diff
@@ -53,6 +53,10 @@ var (
 			Name:  "plain-http",
 			Usage: "allow connections using plain HTTP",
 		},
+		cli.BoolFlag{
+			Name:  "trace",
+			Usage: "enable HTTP tracing for registry interactions",
```

Comment:
Does it have a possibility of leaking secret information into stderr?
If so, can we have a warning text about that in `Usage` field?

Reply:
Right now, because of the tracer object in remotes/docker/resolver.go, it doesn't log sensitive information such as HTTP headers.